config/routes.rb

```diff
@@ -54,40 +58,48 @@
   patch "/profile/orders/:order_id/cancel", to: "user_orders#cancel"
 
   namespace :merchant do
-    get '/', to: "dashboard#show"
-    get "/items", to: "items#index"
-    get 'items/:id/edit', to: "items#edit"
     put "items/:id", to: "items#update_active"
     patch "items/:id", to: "items#update"
-    get "/items/add-item", to: "items#new"
-    post "/items", to: "items#add_item"
+    resources :items, only: [:index, :edit, :new, :create, :destroy]
+    get '/', to: "dashboard#show"
+    # get "/items", to: "items#index"
+    # get 'items/:id/edit', to: "items#edit"
+
+    # get "/items/new", to: "items#new"
+    # post "/items", to: "items#create"
     patch '/:order_id/:item_id', to: "orders#update"
     get "/orders/:id", to: "orders#show"
-    resources :items, only: [:destroy]
+    # resources :items, only: [:destroy]
   end
 
   namespace :admin do
     get '/', to: "dashboard#show"
-    get "/users", to: "users#index"
-    get '/users/:id', to: 'users#show'
+    resources :users, only: [:index, :show]
+    # get "/users", to: "users#index"
+    # get '/users/:id', to: 'users#show'
     patch '/:id', to: 'merchants#update'
-    get '/merchants', to: "merchants#index"
-    get '/merchants/new', to: 'merchants#new'
-    post '/merchants/', to: 'merchants#create'
+
+    # get '/merchants', to: "merchants#index"
+    # get '/merchants/new', to: 'merchants#new'
+    # post '/merchants/', to: 'merchants#create'
     patch '/merchants/:id', to: 'merchants#status'
-    get '/merchants/:id', to: 'merchants#show'
+    # get '/merchants/:id', to: 'merchants#show'
     get '/merchants/:id/items', to: 'merchant_items#index'
-    get '/merchants/:id/edit', to: 'merchants#edit'
+    # get '/merchants/:id/edit', to: 'merchants#edit'
     put '/merchants/:id/', to: 'merchants#update_merchant'
-    delete '/merchants/:id', to: 'merchants#destroy'
-    get '/merchants/:id/items/add-item', to: 'merchant_items#new'
-    post '/merchants/:id/items', to: 'merchant_items#create'
-    get '/merchants/:merchant_id/items/:id/edit', to: 'merchant_items#edit'
-    patch '/merchants/:merchant_id/items/:id', to: 'merchant_items#update'
-    delete '/merchants/:merchant_id/items/:id', to: 'merchant_items#destroy'
-    put '/merchants/:merchant_id/items/:id', to: 'merchant_items#update_active'
-    get '/merchants/:merchant_id/orders/:id', to: 'merchant_orders#show'
-    patch '/merchants/:merchant_id/orders/:order_id/:item_id', to: 'merchant_orders#update'
+    # delete '/merchants/:id', to: 'merchants#destroy'
```

Comment:
Just left in the old routes as comments so others knew which ones the resources affected